src/containers/Homepage/index.js

```diff
@@ -15,21 +16,23 @@ function Homepage() {
 	const [listTodo, setListTodo] = useState([]);
 	const [countSuccess, setCountSuccess] = useState(0);
 	const [clearSuccess, setClearSuccess] = useState(false);
-	const [isLoading, setIsLoading] = useState(false);
+	const [isLoadingData, setIsLoadingData] = useState(false);
 	const inputRef = useRef(null);
 	const todos = useSelector((state) => state.todos);
 	const dispatch = useDispatch();
 
+	const { isLoading } = useSelector((state) => state.loading);
+
 	const handleChangeView = (action) => {
 		setListTodo(action);
 	};
 
 	useEffect(() => {
-		setIsLoading(true);
+		setIsLoadingData(true);
```

Comment:
Note: should calling 'isLoading' from reducer and using in case request API has running

Reply:
Updated!